packer/tests/spec/packages_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'spec_helper'
 
 describe command('java -version') do
-    its(:stderr){ should include 'openjdk version "1.8' }
+  its(:stderr) { should include 'openjdk version "1.8' }
+end
+
+describe command('ps -ef | grep doge-webapp.jar') do
```

Comment:
Can you see if the `doge-webapp` is in the service instead of looking at the running processes? `sudo system doge-webapp status` or something like that.